frontend/cypress/e2e/navbar.cy.js

```diff
@@ -0,0 +1,68 @@
+//cy.signup("name", "surname", "someone@example.com", "password");
+
+
+describe('Navbar', () => {
+  it('shows navbar once user has signed in', () => {
+    cy.visit("/signup")
+    cy.get('nav').should('not.exist')
+    cy.signup("name", "surname", "someone@example.com", "password");
+
+    cy.visit("/login");
+    cy.get('nav').should('not.exist')
+    cy.get("#email").type("someone@example.com");
+    cy.get("#password").type("password");
+    cy.get("#submit").click();
+    cy.wait(1000)
```

Comment:
[suggestion] it seems like this block is repeated in every test. Would it be better if we do a beforeEach block?